scripts/mergehex.py

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env python3
+#
+# Copyright (c) 2018 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: Apache-2.0
+
```

Comment:
add some intro here about this script and what it does.

Reply:
OK like this?